paparazzi/src/main/java/app/cash/paparazzi/accessibility/AccessibilityOverlayDetailsView.kt

```diff
@@ -51,8 +51,8 @@ internal class AccessibilityOverlayDetailsView(context: Context) : FrameLayout(c
     setBackgroundColor(RenderSettings.DEFAULT_DESCRIPTION_BACKGROUND_COLOR.toColorInt())
   }
 
-  fun addElements(elements: Collection<AccessibilityElement>) {
-    accessibilityElements.addAll(elements)
+  fun updateElements(elements: Collection<AccessibilityElement>) {
+    accessibilityElements = elements
```

Comment:
keep `addAll` and clear on each add, if you're looking to avoid leaking memory?  this allows the list to remain a val.